metaflow/cli.py

```diff
@@ -234,8 +234,17 @@ def output_dot(obj):
 )
 @click.pass_obj
 def dump(obj, input_path, private=None, max_value_size=None, include=None, file=None):
-
     output = {}
+    
+    if file is not None and max_value_size is not None:
+        raise CommandException("max_value_size set to None when a file variable is provided will get replace the file variable")
+    elif file is None and max_value_size is None:
+        echo("max_value_size will be set to 1000 when neither file nor max_value_size is set to prevent all information being printed to stdout")
```

Comment:
the default is already set of 1000 in L:220 above - is this needed?

Reply:
Great call out @savingoyal we can remove this if we don't want the redundancy in our code. Let's follow the DRY principle and remove this condition